.travis.yml

```diff
@@ -1,102 +1,97 @@
-language: python
-
 # Run jobs on container-based infrastructure, can be overridden per job
-dist: xenial
 
-# Run builds for both master and topology (and PRs to each)
+# Run builds for both master and feature releases
+
+language: python
+
 branches:
   only:
   - master
   # Regex for feature release branches
-  - /.*[.]0-branch$/
+  - "/.*[.]0-branch$/"
```

Comment:
Release branches are canonically named something like `maintenance/0.4.x` or similar.

Reply:
Thanks for the insight. Updated in 7981ad7